src/booster-firejail.h

```diff
@@ -0,0 +1,56 @@
+/***************************************************************************
+**
+** Copyright (C) 2014 - 2020 Jolla Ltd.
```

Comment:
New derivative work. This line is not needed.

Reply:
Removed from both